nifi-toolkit/nifi-toolkit-assembly/docker/Dockerfile

```diff
@@ -43,7 +43,7 @@ RUN chown -R nifi:nifi ${NIFI_TOOLKIT_BASE_DIR}
 USER nifi
 
 # Default port for TLS Toolkit CA Server
-EXPOSE 8443
+EXPOSE 9443
```

Comment:
The toolkit guide adoc also needs to be updated for 9443.

Reply:
Thanks, good catch.